src/datatypes/colorizer.md

```diff
@@ -2,10 +2,19 @@
 
 A colorizer specifies a mapping between values and pixels/objects of an output image.
 Different variants of colorizers perform different kinds of mapping.
+In general, there are two families of colorizers: **_gradient_** and **_palette_**.
+_Gradients_ are used to interpolate a continuous spectrum of colors between explicitly stated tuples (`breakpoints`) of a value and a color.
+A `palette` colorizer on the other hand, is used to generate a discrete set of colors, each mapped to a specific value.
 
-Usually, there are two miscellaneous fields in each colorizer, namely `noDataColor` and `defaultColor`.
+There are three miscellaneous fields in both of the gradient colorizers, namely `noDataColor`, `overColor` and `underColor`.
 The field `noDataColor` is used for all missing, `NaN` or no data values.
-The `defaultColor` is used for all overflowing values, for instance, if there are breakpoints defined from `0` to `10`, but a value of `-5` or `11` is mapped to a color.
+The fields `overColor` and `underColor` are used for all overflowing values.
+For instance, if there are breakpoints defined from `0` to `10`, but a value of `-5` or `11` is mapped to a color, the respective field will be chosen instead.
+This way, you can specifically highlight values that lie outside of a given range.
+
+For a `palette` colorizer, there are no `overColor` and `underColor` fields.
+If a given value does not match any entry in the palette's definition, it is mapped to the `defaultColor`.
+The `noDataColor` field is used for values, that cannot be mapped, for instance `NaN`.
```

Comment:
```suggestion
The `noDataColor` field is used for no data values or `NaN`.
```

But somehow this is already described in line 10.
So you could as well remove it.

Reply:
994b87e